fair_mango/metrics/metrics.py

```diff
@@ -442,17 +461,21 @@ def __call__(self) -> tuple[Sequence, list]:
                 else:
                     real_values = real_y_group
                     predicted_values = predicted_y_group
+
                 for metric_name, metric in self.metrics.items():
                     result_for_group.setdefault(metric_name, []).append(
                         metric(real_values, predicted_values)
                     )
+
             self.results.append(result_for_group)
+
         return self.data.real_target, self.results
 
 
 def difference(result_per_groups: np.array) -> dict:
```

Comment:
the type is `np.ndarray`. Do not take my word for it, check with google

be more ambitious about your typing. Check if it returns `dict[str, list[float] | float]` or not

Reply:
You are right!